podvm/Makefile.inc

```diff
@@ -32,19 +32,28 @@ endif
 IMAGE_SUFFIX ?= 
 IMAGE_FILE = $(IMAGE_NAME)$(IMAGE_SUFFIX)
 
+LIBC_ARCH := $(shell uname -m)
+LIBC := $(if $(findstring s390x,$(LIBC_ARCH)),gnu,musl)
 
 AGENT_PROTOCOL_FORWARDER = $(FILES_DIR)/usr/local/bin/agent-protocol-forwarder
 KATA_AGENT    = $(FILES_DIR)/usr/local/bin/kata-agent
 PAUSE         = $(FILES_DIR)/$(PAUSE_BUNDLE)/rootfs/pause
 SKOPEO    = $(FILES_DIR)/usr/bin/skopeo
 UMOCI     = $(FILES_DIR)/usr/local/bin/umoci
+ATTESTATION_AGENT = $(FILES_DIR)/usr/local/bin/attestation-agent
 
 BINARIES = $(AGENT_PROTOCOL_FORWARDER) $(KATA_AGENT) $(PAUSE)
 
 ifdef USE_SKOPEO
 BINARIES += $(SKOPEO) $(UMOCI)
 endif
 
+KBC_URI ?= "null"
+ifdef AA_KBC
+BINARIES += $(ATTESTATION_AGENT)
+$(shell sed -i "s/\(aa_kbc_params = \)\"[^\"]*\"/\1\"${AA_KBC}::${KBC_URI}\"/g" $(FILES_DIR)/etc/agent-config.toml)
```

Comment:
Is it `\"${AA_KBC}::${KBC_URI}\"` or `\"${AA_KBC}:${KBC_URI}\"`?

Reply:
It is: ```\"${AA_KBC}::${KBC_URI}\"```